internal/service/eks/addon_test.go

```diff
@@ -912,3 +944,15 @@ resource "aws_eks_addon" "test" {
 }
 `, rName, addonName, tagKey1, tagValue1, tagKey2, tagValue2))
 }
+
+func testAccAddonConfig_configurationValues(rName, addonName, addonVersion, configurationValues string) string {
+	return acctest.ConfigCompose(testAccAddonBaseConfig(rName), fmt.Sprintf(`
+resource "aws_eks_addon" "test" {
+  cluster_name 			= aws_eks_cluster.test.name
+  addon_name   			= %[2]q
+  addon_version     	= %[3]q
+  configuration_values  = %[4]q
+  resolve_conflicts     = "OVERWRITE"
```

Comment:
We might consider AccTests for each of the values of resolve_conficts.
https://docs.aws.amazon.com/eks/latest/APIReference/API_CreateAddon.html#API_CreateAddon_RequestSyntax:~:text=Required%3A%20No-,resolveConflicts,-How%20to%20resolve